hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -196,6 +200,10 @@ public IOType getIOType() {
       stat.setRuntimeStats(runtimeStats);
```

Comment:
We already have these `runtimeStats`, can we use this for write times instead ?

Reply:
reused the timer.